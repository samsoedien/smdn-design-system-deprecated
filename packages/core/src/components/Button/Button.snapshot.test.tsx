import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer' // eslint-disable-line
import 'jest-styled-components'

const Button = styled.button`
  color: blue;
`

test('Button snapshot works', () => {
  const tree = renderer.create(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
})
