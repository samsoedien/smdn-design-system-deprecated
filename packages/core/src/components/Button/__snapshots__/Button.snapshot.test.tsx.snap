// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot works 1`] = `
<button
  className=""
/>
`;
